### DISCUSSION TITLE
Add more policies examples

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#1216] [Feature Requests]
Hi 👋🏻 
At the moment I am very annoyed because I would like to make the bet to use supabase as a backend directly. The problem is that I would need to make sure that a normal user does not have access to the same data as another user, and the only way seems to be through policies.
Unfortunately, I find that there is a lack of policy examples to understand them properly, because it seems that it is very complex.

Would it be possible to add more?

**Comment by steve-chavez** [comment]
Hey @Olyno,

Did you had a look at https://supabase.io/docs/guides/auth#policy-examples ?

What type of example do you think it's missing? Perhaps you have a sample schema you can share?

**Reply by Olyno** [reply]
Hi 👋🏻 
Well, the examples in the documentation show how to use policies in queries, but we see very little about how to use them via the Supabase interface. I think it would be a plus to be able to be guided directly on the interface and to have concrete examples like having a project with a role system like "admin", "moderator" etc... and see how with Supabase we can restrict the data.

**Comment by Olyno** [comment]
Done in the last community video, coming soon everywhere: https://youtu.be/qETcl3SUfzU?t=183 🎉 🚀
